packages/superset-ui-legacy-preset-chart-nvd3/src/utils.js

```diff
@@ -143,7 +143,7 @@ export function generateAreaChartTooltipContent(d, timeFormatter, valueFormatter
     "<table><thead><tr><td colspan='4'>" +
     `<strong class='x-value'>${timeFormatter(d.value)}</strong>` +
     '</td></tr></thead><tbody>' +
-    '<tr><td></td><td>Category</td><td>Value</td><td>% to total</td></tr>';
+    '<tr class="legend-header"><td></td><td>Category</td><td>Value</td><td>% to total</td></tr>';
```

Comment:
this should probably be `tooltip-header` or maybe `tooltip-column-names`? I don't think it's technically a legend

Reply:
Agreed! `tooltip-header` it is!